mica/quaternion.py

```diff
@@ -101,6 +102,7 @@ class Quat(object):
 
     """
     def __init__(self, attitude, intype=None):
+        warnings.warn("mica.quaternion is deprecated as of Quaternion 3.5.0", FutureWarning)
```

Comment:
I don't know if adding the warning just causes more trouble for ourselves, but seems fine in this application.